### ISSUE TITLE
Admin part - how to use custom fields

### ISSUE THREAD

**Issue by Jimmi08** [Jimmi08/glossary#6]
1. install Custom Field plugin
![image](https://user-images.githubusercontent.com/5429548/31543293-b656365e-b015-11e7-9723-bca977004555.png)

2. table glossary has to have field 'fields'  or any other we will agree on
 
![image](https://user-images.githubusercontent.com/5429548/31543374-0bb441cc-b016-11e7-9c46-de7f1dc8d0cc.png)

3. add this table to CF plugin
- set tab title
- field names, labels and types (and parameters f.e. for media category)
 
![image](https://user-images.githubusercontent.com/5429548/31543386-14d48410-b016-11e7-84d3-6a83b74cff0b.png)

4. set values for your words
 
![image](https://user-images.githubusercontent.com/5429548/31543393-1df98df6-b016-11e7-9ae3-288bc2c4fd56.png)

**Comment by rica-carv** [comment]
It will also require a custom theme shortcode for image output on custom glossary template, right?

**Comment by Jimmi08** [comment]
Yes, or maybe I will be able to do global shortcodes.  Not so easy as I thought. I can't get it loaded in glossary plugin.

**Comment by rica-carv** [comment]
> I can't get it loaded in glossary plugin.

Maybe because glossary is still a mix of old code with v2 one....

**Comment by Jimmi08** [comment]
You cant, you havent that plugin yet
